+sigopt/Connection.m

```diff
@@ -0,0 +1,22 @@
+classdef Connection
+  properties
+    client_token
```

Comment:
yeah - it's `client_token` in the python client for legacy reasons but since this is a new client it seems safe to set as api token now. will fix